pkg/cli/flags/connect/connect.go

```diff
@@ -22,6 +22,7 @@ func AddCommonFlags(cmd *cobra.Command, options *cli.ConnectOptions) {
 	cmd.Flags().IntVar(&options.ServiceAccountExpiration, "token-expiration", 0, "If specified, vCluster will create the service account token for the given duration in seconds. Defaults to eternal")
 	cmd.Flags().BoolVar(&options.Insecure, "insecure", false, "If specified, vCluster will create the kube config with insecure-skip-tls-verify")
 	cmd.Flags().BoolVar(&options.BackgroundProxy, "background-proxy", true, "Try to use a background-proxy to access the vCluster. Only works if docker is installed and reachable")
+	cmd.Flags().StringVar(&options.BackgroundProxyImage, "background-proxy-image", "bitnami/kubectl:1.29", "The image to use for the background proxy. Only used if --background-proxy is enabled.")
```

Comment:
We should push this into a constant so we only have to do in one place. Maybe @jjaferson can help